src/main/java/bio/terra/cli/command/auth/Login.java

```diff
@@ -11,6 +12,11 @@
     description = "Authorize the CLI to access Terra APIs and data with user credentials.")
 public class Login extends BaseCommand {
 
+  @CommandLine.Option(
+      names = "--use-app-default-credentials",
+      description = "Use application default credentials to log into terra CLI instead of browser.")
+  private boolean useAppDefaultCreds;
```

Comment:
If this flag isn't specified, does the value default to false?

Reply:
yes